src/main/kotlin/Stores/ChannelSubscriptionStore.kt

```diff
@@ -0,0 +1,88 @@
+package Stores
+
+import models.ChannelSubscription
+import io.ktor.http.Parameters
+import models.Channel
+import models.User
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.insert
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.selectAll
+import org.joda.time.DateTime
+import java.util.*
+data class UsersSmallObj(
```

Comment:
why the `Small` naming convention here?

Reply:
also, let's keep the `Users` name singular to `User` since it'll be referring to just one